.github/workflows/build_site.yml

```diff
@@ -3,22 +3,28 @@ name: Construir el sitio con Nikola
 on:
   pull_request:
     branches: [ master ]
+    paths: 
+      - conf.py
+      - pages/**
+      - themes/**
+      - plugins/**
+      - images/**
 
 jobs:
   build:
 
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v2
-    - name: Set up Python 3.8
-      uses: actions/setup-python@v1
+    - uses: actions/checkout@v4
+    - name: Set up Python 3.11
+      uses: actions/setup-python@v5
       with:
-        python-version: 3.8
+        python-version: 3.11
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        pip install -r requirements_freeze.txt
+        pip install -r requirements.txt
```

Comment:
why not using the freeze version? Should we delete it if we are not planning to keep using ?

Reply:
Yes, I remove it in a follow up PR https://github.com/PyAr/wiki/pull/153